packages/js/data/src/payment-gateway-suggestions/resolvers.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getPaymentGatewaySuggestionsSuccess,
+	getPaymentGatewaySuggestionsError,
+	getPaymentGatewaySuggestionsRequest,
+	getOfflinePaymentGatewaysRequest,
+	getOfflinePaymentGatewaysSuccess,
+	getOfflinePaymentGatewaysError,
+} from './actions';
+import { PaymentGatewaySuggestionsState } from './types';
+import { WC_ADMIN_NAMESPACE } from '../constants';
+
+export function* getPaymentGateways() {
+	yield getPaymentGatewaySuggestionsRequest();
+
+	try {
+		const paymentGatewaySuggestions: PaymentGatewaySuggestionsState =
+			yield apiFetch( {
+				path: WC_ADMIN_NAMESPACE + '/settings/payments/providers',
+			} );
+		yield getPaymentGatewaySuggestionsSuccess( paymentGatewaySuggestions );
+	} catch ( e ) {
+		yield getPaymentGatewaySuggestionsError( e );
+	}
+}
+
+export function* getOfflinePaymentGateways() {
+	yield getOfflinePaymentGatewaysRequest();
+
+	try {
+		const paymentGatewaySuggestions: PaymentGatewaySuggestionsState =
+			yield apiFetch( {
+				path: WC_ADMIN_NAMESPACE + '/settings/payments/providers',
```

Comment:
This looks identical to the above 👀

Reply:
Yes, I fixed it in https://github.com/woocommerce/woocommerce/pull/52824/commits/69f51b99ac41b32f4f67bbea89de6ddd68d2791e